nightingale_dispatcher/src/nightingale_dispatcher/move_arm_task.py

```diff
@@ -9,8 +9,20 @@
 class MoveArmTask(Task):
     def __init__(self):
         self.manipulation = ManipulationControl()
+        if not self.manipulation.home_left() or not self.manipulation.home_right():
+            rospy.logfatal("MoveArmTask: Failed to home arms. This is unrecoverable")
 
-    def execute(self, end_pose):
-        assert len(end_pose) is 7
-        self.manipulation.jnt_ctrl.cmd_right_arm(end_pose)
-        return TaskCodes.SUCCESS
+    def extend_restock(self):
+        result = self.manipulation.extend_restock()
+        assert type(result) is bool
+        return TaskCodes.SUCCESS if result else TaskCodes.ERROR
+
+    def retract_right_arm(self):
+        result = self.manipulation.home_right()
```

Comment:
- [ ] home_right() currently closes the right gripper. this function should just retract the right arm back to home